src/content/learn/managing-state.md

```diff
@@ -336,7 +335,6 @@ const contacts = [
 
 ```js ContactList.js
 export default function ContactList({
-  selectedContact,
```

Comment:
Instead of removing this prop, can we add bold-en the selected contact name in this example? I think that will make this example clearer when you've selected a name.